components/channel_header_dropdown/menu_items/toggle_members_rhs/toggle_members_rhs.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {Channel} from '@mattermost/types/channels';
+import Menu from 'components/widgets/menu/menu';
+
+type Action = {
+    closeRightHandSide: () => void;
+    showChannelMembers: (channelId: string, editMembers: boolean) => void;
+};
+
+type OwnProps = {
+    channel: Channel;
+    show: boolean;
+    id: string;
+    editMembers?: boolean;
+    textOpen: string;
+    textClose: string;
+}
+
+type Props = {
+    rhsOpen: boolean;
+    actions: Action;
+} & OwnProps;
+
+const ToggleChannelMembersRHS = ({
```

Comment:
Should we add a cypress test to cover this behavior?

Reply:
Yes that's a good catch, I've changed so much e2e tests covering this menu option (both View and Manage members) that I didn't think about creating dedicated tests

Reply:
change applied: I added a test case for both 'View Members' and 'Manage Members', and also added a missing case for clicking on the 'Members icon' in the channel header. Thanks for suggesting those changes!